src/components/Header/index.jsx

```diff
@@ -41,8 +41,12 @@ class Header extends Component {
     return pathname.indexOf(curLink) > -1;
   };
 
-  becomeDeveloper = () => {
-    setCookie('role', '', -1);
+  becomeDeveloper = isNormal => {
+    if (isNormal) {
```

Comment:
you can use `user provider`'s method

Reply:
The method for add cookie to role when changed normal view, or remove cookie when back developer view.